src/saleor_app/workers/sqs.py

```diff
@@ -0,0 +1,117 @@
+import asyncio
+import logging
+from typing import Any, List, Union
+
+import boto3
+from pydantic import ValidationError
+
+from saleor_app.settings import AWSSettings
+from saleor_app.workers.base import SaleorAppWorker
+from saleor_app.workers.errors import (
+    NonTransientError,
+    TransientError,
+    UnrecognizedEventPayload,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class SaleorAppSQSWorker(SaleorAppWorker):
+    def __init__(
+        self,
+        queue_name: str,
+        max_number_of_messages: int = 1,
+        wait_time_seconds: int = 0,
+        *args,
+        **kwargs,
+    ):
+        super().__init__(*args, **kwargs)
+        self.transient_error_wait_seconds = 5
```

Comment:
You have some configuration hardcoded, and some recieved from `__init__`. Wouldn't be good to unify this somehow? IMO all of this data should come from one place - like SQS settings class or something